pkg/network/ebpf/c/prebuilt/tracer.c

```diff
@@ -56,6 +57,9 @@ int kprobe__tcp_sendmsg(struct pt_regs* ctx) {
 
     handle_tcp_stats(&t, skp);
     get_tcp_segment_counts(skp, &packets_in, &packets_out);
+    if (t.dport == 22) {
```

Comment:
Is there a reason you are doing this in eBPF? It would be much simpler to do it in userspace, especially since eBPF doesn't deal with strings well.

Reply:
It's more for testing, I sent you a msg on slack to talk about that later (1/1)